### PR TITLE
Fix making zh-cn translation more readable

### PR DESCRIPTION
**Pull request by BruceZhang1993**
I fixed zh-cn translation for all desktop files. but left the scripts unchanged because it's not made to support multi-lang. so  that's it

### CODE REVIEW

File: ServiceMenus/Android-Tools_servicemenu.desktop [hunk 1]
```diff
@@ -79,5 +79,5 @@ Name[ko]=I재시동의 관리자 Android
 Name[pt]=Administrador de Mim Reinicia para Android
 Name[ru]=Android Reboot Менеджер
 Name[uk]=Android Reboot Менеджер
-Name[zh_CN]=我的管理人重新开始 Android
+Name[zh_CN]=Android 电源管理器
```
Comment (rtlanceroad) [inline ServiceMenus/Android-Tools_servicemenu.desktop]: Why it is " 电源管理器" ? It should be something like " 启动管理器"。I checked that shell scripts in `Exec` line, It dealt with bootloader stuff, so it should be "启动管理器".

File: ServiceMenus/Change-Owner_addtoservicemenu.desktop [hunk 2]
```diff
@@ -21,5 +21,5 @@ Name[ko]=소유자를 여기에서 바꾼다
 Name[pt]=Mudar o Proprietário Aqui
 Name[ru]=Изменить здесь владельца
 Name[uk]=Змінити тут власника
-Name[zh_CN]=在这里改变所有者
+Name[zh_CN]=改变文件所有者
```
Comment (rtlanceroad) [inline ServiceMenus/Change-Owner_addtoservicemenu.desktop]: "改变所有人信息"

File: ServiceMenus/CheckSum-Tools_servicemenu.desktop [hunk 3]
```diff
@@ -32,7 +32,7 @@ Name[ko]=체크섬을 증명한다
 Name[pt]=Verificar CheckSum
 Name[ru]=Проверить контрольные суммы
 Name[uk]=Перевірити контрольні суми
-Name[zh_CN]=查证核对和
+Name[zh_CN]=验证校验和
```
Comment (rtlanceroad) [inline ServiceMenus/CheckSum-Tools_servicemenu.desktop]: "校验" is enough.

File: ServiceMenus/Compressed-File-Integrity-Check_addtoservicemenu.desktop [hunk 4]
```diff
@@ -21,5 +21,5 @@ Name[ko]=파일 알약의 성실의 검사
 Name[pt]=Exame de Integridade de Tablete de Arquivo
 Name[ru]=Сжал Проверку Целостности Файла
 Name[uk]=Стиснув Перевірку Цілісності Файлу
-Name[zh_CN]=文件平板的正直审查
+Name[zh_CN]=压缩包完整性检查
```
Comment (rtlanceroad) [inline ServiceMenus/Compressed-File-Integrity-Check_addtoservicemenu.desktop]: "压缩文件完整性检查"

File: ServiceMenus/Dropbox-Tools_servicemenu.desktop [hunk 5]
```diff
@@ -62,7 +62,7 @@ Name[ko]=Dropbox 대중을 복사하고, URL를 얻는다
 Name[pt]=Copiar o Público de Dropbox e obter URL
 Name[ru]=Копировать в Dropbox с общим доступом и получить ссылку
 Name[uk]=Копіювати до Dropbox із публічним доступом і отримати посиланяя
-Name[zh_CN]=复印 Dropbox 民众并且获得网址
+Name[zh_CN]=复制到 Dropbox 并分享
```
Comment (rtlanceroad) [inline ServiceMenus/Dropbox-Tools_servicemenu.desktop]: "复制到 Dropbox 以及 获取衔接"

File: ServiceMenus/Dropbox-Tools_servicemenu.desktop [hunk 6]
```diff
@@ -77,7 +77,7 @@ Name[ko]=Dropbox 대중을 움직이고, URL를 얻는다
 Name[pt]=Se Mudar para Público de Dropbox e obter URL
 Name[ru]=Переместить в Dropbox с общим доступом и получить ссылку
 Name[uk]=Пересунути до Dropbox із публічним доступом і отримати посиланяя
-Name[zh_CN]=移动到 Dropbox 民众并且获得网址
+Name[zh_CN]=移动到 Dropbox 并分享
```
Comment (rtlanceroad) [inline ServiceMenus/Dropbox-Tools_servicemenu.desktop]: "移动到 Dropbox 以及获取衔接"

File: ServiceMenus/Android-Tools_servicemenu.desktop (hunk 1 above)
Comment (BruceZhang1993) [inline ServiceMenus/Android-Tools_servicemenu.desktop]: "启动管理器" means startup manager which should manage what will start at boot. So the former seems better.

File: ServiceMenus/Dropbox-Tools_servicemenu.desktop [hunk 7]
```diff
@@ -167,4 +167,4 @@ Name[ko]=서비스 Dropbox 의 안에서 올라가고 있는 출발을 가능하
 Name[pt]=Habilitar começos para cima do serviço Dropbox
 Name[ru]=Автозапуск сервиса Dropbox
 Name[uk]=Автозапуск сервіса Dropbox
-Name[zh_CN]=服务 Dropbox 使开始能够增加
+Name[zh_CN]=自动启动 Dropbox 服务
```
Comment (rtlanceroad) [inline ServiceMenus/Dropbox-Tools_servicemenu.desktop]: “启用自动启动 Dropbox 服务”

File: ServiceMenus/ISO-Image-Tools_servicemenu.desktop [hunk 8]
```diff
@@ -80,7 +80,7 @@ Name[ko]=MD5sum 이미지 iso9660를 보여준다
 Name[pt]=Mostrar Imagem de MD5sum o iso9660
 Name[ru]=Показ kонтрольная сумма MD5 iso9660-образа
 Name[uk]=Покажіть kонтрольна сума MD5 iso9660-образу
-Name[zh_CN]=表示 MD5sum 图像 iso9660
+Name[zh_CN]=查看 ISO9660 镜像 MD5 校验和
```
Comment (rtlanceroad) [inline ServiceMenus/ISO-Image-Tools_servicemenu.desktop]: "查看 ISO9660 镜像MD5值"

File: ServiceMenus/ISO-Image-Tools_servicemenu.desktop [hunk 9]
```diff
@@ -96,7 +96,7 @@ Name[ko]=SHA1sum 이미지 iso9660를 보여준다
 Name[pt]=Mostrar Imagem de SHA1sum o iso9660
 Name[ru]=Показ kонтрольная сумма SHA1 iso9660-образа
 Name[uk]=Покажіть kонтрольна сума SHA1 iso9660-образу
-Name[zh_CN]=表示 SHA1sum 图像 iso9660
+Name[zh_CN]=查看 ISO9660 镜像 SHA1 校验和
```
Comment (rtlanceroad) [inline ServiceMenus/ISO-Image-Tools_servicemenu.desktop]: “查看 ISO9660 镜像 SHA1值”’

File: ServiceMenus/ISO-Image-Tools_servicemenu.desktop [hunk 10]
```diff
@@ -112,7 +112,7 @@ Name[ko]=SHA256sum 이미지 iso9660를 보여준다
 Name[pt]=Mostrar Imagem de SHA256sum o iso9660
 Name[ru]=Показ kонтрольная сумма SHA256 iso9660-образа
 Name[uk]=Покажіть kонтрольна сума SHA256 iso9660-образу
-Name[zh_CN]=表示 SHA256sum 图像 iso9660
+Name[zh_CN]=查看 ISO9660 镜像 SHA256 校验和
```
Comment (rtlanceroad) [inline ServiceMenus/ISO-Image-Tools_servicemenu.desktop]: "查看 ISO9660 镜像SHA256值"

File: ServiceMenus/ISO-Image-Tools_servicemenu.desktop [hunk 11]
```diff
@@ -128,7 +128,7 @@ Name[ko]=SHA512sum 이미지 iso9660를 보여준다
 Name[pt]=Mostrar Imagem de SHA512sum o iso9660
 Name[ru]=Показ kонтрольная сумма SHA512 iso9660-образа
 Name[uk]=Покажіть kонтрольна сума SHA512 iso9660-образу
-Name[zh_CN]=表示 SHA512sum 图像 iso9660
+Name[zh_CN]=查看 ISO9660 镜像 SHA512 校验和
```
Comment (rtlanceroad) [inline ServiceMenus/ISO-Image-Tools_servicemenu.desktop]: "查看 ISO9660 镜像SHA512值"

File: ServiceMenus/Multimedia-Tools_servicemenu.desktop [hunk 12]
```diff
@@ -175,7 +175,7 @@ Name[ko]=붙는 것은 파일 MP3 에 커버했다
 Name[pt]=Prender Coberto a Arquivos MP3
 Name[ru]=Приложите Конверт к MP3 Файлы
 Name[uk]=Додайте Обкладинку MP3 Файли
-Name[zh_CN]=附上复盖到文件 MP3
+Name[zh_CN]=添加 MP3 文件封面图片
```
Comment (rtlanceroad) [inline ServiceMenus/Multimedia-Tools_servicemenu.desktop]: "给MP3文件添加封面"。 Your translation changed the original meaning.

File: ServiceMenus/PDF-Tools_servicemenu.desktop [hunk 13]
```diff
@@ -32,7 +32,7 @@ Name[ko]=DRM를 적용한다 암호화한다
 Name[pt]=Aplique DRM + Codifique
 Name[ru]=Задействовать DRM + Шифрование
 Name[uk]=Задіяти DRM + Шифрування
-Name[zh_CN]=应用 DRM 编加密码
+Name[zh_CN]=DRM 及加密
```
Comment (rtlanceroad) [inline ServiceMenus/PDF-Tools_servicemenu.desktop]: "应用DRM流及加密"

File: ServiceMenus/PDF-Tools_servicemenu.desktop [hunk 14]
```diff
@@ -128,5 +128,5 @@ Name[ko]=사용자 암호 (암호화한다)를 적용해라
 Name[pt]=Aplique Contra-senha de Usuário (Codifique)
 Name[ru]=Задействовать пароль пользователя (Зашифровать)
 Name[uk]=Задіяти пароль користувача (Зашифрувати)
-Name[zh_CN]=应用使用者密码 (编加密码)
+Name[zh_CN]=添加拥有者密码 (加密)
```
Comment (rtlanceroad) [inline ServiceMenus/PDF-Tools_servicemenu.desktop]: "添加用户密码 (加密)"。The user here means 3rd party people like you and me.

File: ServiceMenus/SSH-Tools_servicemenu.desktop [hunk 15]
```diff
@@ -96,7 +96,7 @@ Name[ko]=멀리 떨어지는 디렉토리에 오른다
 Name[pt]=Montar Diretório Remoto
 Name[ru]=Смонтировать удалённую папку
 Name[uk]=Змонтувати віддалену теку
-Name[zh_CN]=装目录遥远的
+Name[zh_CN]=挂载远程目录
```
Comment (rtlanceroad) [inline ServiceMenus/SSH-Tools_servicemenu.desktop]: "挂载到远程目录"

File: ServiceMenus/SaMBa-Tools_servicemenu.desktop [hunk 16]
```diff
@@ -31,5 +31,5 @@ Name[ko]=분담했던 SaMBa 링커
 Name[pt]=Conector de ter Compartilhado SaMBa
 Name[ru]=Монтировать сетевые ресурсы SaMBa
 Name[uk]=Монтувати мережеві ресурси SaMBa
-Name[zh_CN]=分享的有的连接器 SaMBa
+Name[zh_CN]=Samba 分享挂载
```
Comment (rtlanceroad) [inline ServiceMenus/SaMBa-Tools_servicemenu.desktop]: "Samba分享挂载器"

File: ServiceMenus/Search-Tools_servicemenu.desktop [hunk 17]
```diff
@@ -79,7 +79,7 @@ Name[ko]=검색 데이터 베이스를 개량한다
 Name[pt]=Atualizar Banco de Dados de Procura
 Name[ru]=Обновить поисковую База данных
 Name[uk]=Оновити пошукову База даних
-Name[zh_CN]=升级搜寻数据库
+Name[zh_CN]=刷新搜索数据库
```
Comment (rtlanceroad) [inline ServiceMenus/Search-Tools_servicemenu.desktop]: No need to change.

File: applications/Android_Tools-reboot.desktop [hunk 18]
```diff
@@ -12,7 +12,7 @@ Name[ko]=I재시동의 관리자 Android
 Name[pt]=Administrador de Mim Reinicia para Android
 Name[ru]=Android Reboot Менеджер
 Name[uk]=Android Reboot Менеджер
-Name[zh_CN]=我的管理人重新开始 Android
+Name[zh_CN]=Android 电源管理器
```
Comment (rtlanceroad) [inline applications/Android_Tools-reboot.desktop]: I still think "Android启动管理器" is better.

File: applications/Dropbox_Tools-service-autostart.desktop [hunk 19]
```diff
@@ -12,7 +12,7 @@ Name[ko]=서비스 Dropbox 의 안에서 올라가고 있는 출발을 가능하
 Name[pt]=Habilitar começos para cima do serviço Dropbox
 Name[ru]=Автозапуск сервиса Dropbox
 Name[uk]=Автозапуск сервіса Dropbox
-Name[zh_CN]=服务 Dropbox 使开始能够增加
+Name[zh_CN]=自动启动 Dropbox 服务
```
Comment (rtlanceroad) [inline applications/Dropbox_Tools-service-autostart.desktop]: "启用自动启动 Dropbox 服务"

File: applications/SSH_Tools-connect-remote-machine.desktop [hunk 20]
```diff
@@ -12,7 +12,7 @@ Name[ko]=먼 하인을 접속한다
 Name[pt]=Conectar o Servidor Remoto
 Name[ru]=Подключиться к удалённому серверу
 Name[uk]=Підключитися до віддаленого серверу
-Name[zh_CN]=连接遥远的仆人
+Name[zh_CN]=连接远程服务器
```
Comment (rtlanceroad) [inline applications/SSH_Tools-connect-remote-machine.desktop]: "连接至远程服务器"

File: applications/SSH_Tools-sshfs.desktop [hunk 21]
```diff
@@ -12,7 +12,7 @@ Name[ko]=멀리 떨어지는 디렉토리에 오른다
 Name[pt]=Montar Diretório Remoto
 Name[ru]=Смонтировать удалённую папку
 Name[uk]=Змонтувати віддалену теку
-Name[zh_CN]=装目录遥远的
+Name[zh_CN]=挂载远程目录
```
Comment (rtlanceroad) [inline applications/SSH_Tools-sshfs.desktop]: "挂载至远程目录"

File: applications/SaMBa_Tools-mount-umount-share.desktop [hunk 22]
```diff
@@ -12,7 +12,7 @@ Name[ko]=분담했던 SaMBa 링커
 Name[pt]=Conector de ter Compartilhado SaMBa
 Name[ru]=Монтировать сетевые ресурсы SaMBa
 Name[uk]=Монтувати мережеві ресурси SaMBa
-Name[zh_CN]=分享的有的连接器 SaMBa
+Name[zh_CN]=Samba 分享挂载
```
Comment (rtlanceroad) [inline applications/SaMBa_Tools-mount-umount-share.desktop]: "Samba分享挂载器"

File: applications/Search_Tools-update-search-db.desktop [hunk 23]
```diff
@@ -12,7 +12,7 @@ Name[ko]=검색 데이터 베이스를 개량한다
 Name[pt]=Atualizar Banco de Dados de Procura
 Name[ru]=Обновить поисковую База данных
 Name[uk]=Оновити пошукову База даних
-Name[zh_CN]=升级搜寻数据库
+Name[zh_CN]=刷新搜索数据库
```
Comment (rtlanceroad) [inline applications/Search_Tools-update-search-db.desktop]: No need to change here.

File: ServiceMenus/Android-Tools_servicemenu.desktop (hunk 1 above)
Comment (rtlanceroad) [inline ServiceMenus/Android-Tools_servicemenu.desktop]: Fair enough.

File: ServiceMenus/Dropbox-Tools_servicemenu.desktop (hunk 7 above)
Comment (BruceZhang1993) [inline ServiceMenus/Dropbox-Tools_servicemenu.desktop]: don't use two verbs in one sentence in chinese. so.

File: ServiceMenus/SSH-Tools_servicemenu.desktop (hunk 15 above)
Comment (BruceZhang1993) [inline ServiceMenus/SSH-Tools_servicemenu.desktop]: what's the difference?

File: ServiceMenus/Dropbox-Tools_servicemenu.desktop (hunk 7 above)
Comment (rtlanceroad) [inline ServiceMenus/Dropbox-Tools_servicemenu.desktop]: The word `Enable` is the thing to be emphasized, So "启用自动启动 Dropbox 服务" or "允许自动启动 Dropbox 服务" expresses same thing in our culture. To be fair, that is not critical, so you decide it.

File: ServiceMenus/SSH-Tools_servicemenu.desktop (hunk 15 above)
Comment (rtlanceroad) [inline ServiceMenus/SSH-Tools_servicemenu.desktop]: Your translation means that "远程目录" is an object(挂载的对象), while the real meaning is it's a destination.

**Pull request merged**